src/Containers/BlogPost.js

```diff
@@ -131,46 +124,55 @@ export default function BlogHome() {
     </SyntaxHighlighter>
   );
 
-  const githubCounterEmojiSelect = (emoji) => {
+  const githubCounterEmojiSelect = emoji => {
     console.log(emoji);
-  }
+  };
 
   const githubCounterAddReaction = () => {
     setAddReaction(!addReaction);
-  }
+  };
 
   useEffect(() => {
     if (!loading) {
       if (error) {
-        console.error(error)
+        console.error(error);
       }
 
       if (data) {
-        const issues = data.repository.issue
+        const issues = data.repository.issue;
         setPost(issues);
         setReactionFun(issues.reactions.nodes);
       }
     }
   }, [loading, error, data, setReactionFun]);
 
   if (loading) {
-    return <Loader />
+    return <Loader />;
   }
 
+  const onBackClick = () => {
+    window.location.replace("#/");
```

Comment:
```suggestion
    window.location.replace(document.referrer);
```
https://developer.mozilla.org/en-US/docs/Web/API/Document/referrer